main.js

```diff
@@ -3,67 +3,30 @@ var saveButton = document.querySelector('.save-btn');
 var displayStarredButton = document.querySelector('.show-starred-button');
 var addCommentButton = document.querySelector('.comment-button');
 var deleteIdeaButton = document.querySelector('.img-button')
-var ideaTitle = document.querySelector('.form-title'); // title text input
-var ideaBody = document.querySelector('.form-body'); // body text input
+var ideaTitle = document.querySelector('.form-title');
+var ideaBody = document.querySelector('.form-body');
 var searchText = document.querySelector('.search-field');
 var renderedIdeas = document.querySelector('.ideabox');
-var savedIdeas = [];
 
+var savedIdeas = [];
+console.log(savedIdeas);
```

Comment:
Remove this console.log

PR's should not contain them. I have this issue myself. I find best way to solve is to just keyword search in your text editor before you push

Reply:
Thank you for the feedback on this one!